src/main/webapp/app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.scss

```diff
@@ -0,0 +1,60 @@
+.edit-sqi {
+    .grading-instructions-update-border {
+        padding: 10px;
+        border-radius: 0.1em;
+        border-width: 1px;
+        border-style: solid;
+        border-color: lightgrey;
```

Comment:
lightgray would be better here. https://www.w3schools.com/colors/colors_shades.asp

Reply:
is there any difference between `lightgrey` and `lightgray`?

Reply:
lightgray is more generally supported